java/ditaa/Portfile

```diff
@@ -1,62 +1,49 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                ditaa
-version             0.9
-revision            3
+github.setup        stathissideris ditaa 0.11.0 v
 categories          java editors
 platforms           darwin
-supported_archs     noarch
 license             GPL-2+
-maintainers         nomaintainer
 
-description         ditaa is a small command-line utility written in Java \
-                    that can convert diagrams drawn using ascii art
+maintainers         {gmail.com:slewsys @slewsys} openmaintainer
 
-long_description    ${description}
+description         Java ASCII art to bitmap graphics conversion utility
 
-homepage            http://ditaa.sourceforge.net
-master_sites        sourceforge
-distfiles           ${name}[strsed ${version} {/\./_/}]-src.zip
-use_zip             yes
+long_description    \
+    ditaa is a small command-line utility, written in Java, that can convert \
+    diagrams drawn using ASCII art (i.e., 'drawings' that contain characters \
+    that resemble lines like | / - ) into proper bitmap graphics.
 
-checksums           md5     d7230273bf4c28c5029d350842278cf9 \
-                    sha1    570893b57cb29efbc919c7a36f119a0db11c46ce \
-                    rmd160  84222a7ded4680d3d0b2b7fac79e19c8983c6233
+github.tarball_from releases
 
-depends_build       bin:ant:apache-ant
+distname            ${name}-${version}-standalone.jar
 
+checksums           md5     0c5e9e1ab720ee18c66bcfeb121512b1 \
```

Comment:
Please remove the md5 and sha1 checksums.

Reply:
Done.  Thank you!

Reply:
Thanks